spp-001-nl mhq heide (#69)
inbo/protocolsource: source/spp/mhq/spp_001_nl_mhq_heide/02_onderwerp.Rmd

-->
```
-   habitattype 2310: droge heide op landduinen
-   habitattype 2330_bu: buntgrasvegetatie op landduinen
-   habitattype 4010: vochtige en natte heide
-   habitattype 4030: droge heide
-   SVI: staat van instandhouding
-   LSVI: lokale staat van instandhouding
-   SBZ: speciale beschermingszone
-   RTK GNSS: Real Time Kinematic Global Navigation Satellite System waarbij cm of decimeter nauwkeurige positiebepaling mogelijk is. Een alternatief is werken met een Fieldmap installatie.
-   Fieldmap: geïntegreerde tool die is ontworpen voor veldgegevensverzameling en ondermeer gebruikt wordt bij de Vlaamse Bosinventaris. De positiebepaling gebeurt hier d.m.v. lasermetingen

## Doelstelling en toepassingsgebied

```{=html}
<!--
Doelstelling: omschrijving in woorden waarvoor het protocol dient.
Voorbeelden wat er met de gegevens kan gedaan worden.
Probeer de doelstelling zo veel mogelijk generiek te beschrijven en vermijd projectspecifieke doelstellingen.
Toepassingsgebied: bespreek hier onder welke condities het protocol kan gebruikt worden.
-->
```
### Doelstelling

> Dit protocol beschrijft de werkwijze die dient gevolgd te worden voor de gegevensinzameling ten behoeve van de 6-jaarlijkse rapportage aan de EU omtrent de staat van instandhouding van de heidehabitats, meer bepaald de habitattypes 2310, 2330_bu, 4010 en 4030.

Reviewer: Dit protocol beschrijft de werkwijze die dient gevolgd te worden voor de gegevensinzameling ten behoeve van de 6-jaarlijkse rapportage aan de EU over de staat van instandhouding (SVI) van de heidehabitats, meer bepaald de habitattypes 2310, 2330_bu, 4010 en 4030.

> Dit protocol beschrijft de werkwijze die dient gevolgd te worden voor de gegevensinzameling ten behoeve van de 6-jaarlijkse rapportage aan de EU omtrent de staat van instandhouding van de heidehabitats, meer bepaald de habitattypes 2310, 2330_bu, 4010 en 4030.

Reviewer: ```suggestion
Dit protocol beschrijft de werkwijze die dient gevolgd te worden voor de gegevensinzameling ten behoeve van de 6-jaarlijkse rapportage aan de EU over de Staat Van Instandhouding (SVI) van de heidehabitats, meer bepaald de habitattypes 2310, 2330_bu, 4010 en 4030.
```

> In deze vegetatietypes dienen in Vlaanderen een 1000-tal random geselecteerde meetpunten in heidehabitat opgemeten te worden en dit telkens op twee verschillende schaalniveaus.
> Enerzijds een vegetatie-opname op een vierkante vegetatieplot van 3 op 3 meter en anderzijds een cirkelvormige plot met diameter 18m waar een aantal bijkomende indicatoren dienen ingeschat te worden.

Reviewer: ```suggestion
In deze vegetatietypes dienen in Vlaanderen een 1000-tal random geselecteerde meetpunten in heidehabitat opgemeten te worden en dit telkens op twee verschillende schaalniveaus:

- een vierkante vegetatieplot van 3 op 3 meter waar een vegetatie-opname wordt uitgevoerd;
- een cirkelvormige plot met diameter 18m waar een aantal bijkomende indicatoren dienen ingeschat te worden.
```

> De meetpunten worden bemeten met een frequentie van 1 keer om de 6 jaar en laten op die manier toe om een uitspraak te doen over het aandeel habitat dit zich in een gunstige/ongunstige toestand bevindt en dit voor elk afzonderlijk habitattype en op niveau Vlaanderen, met een hogere nauwkerigheid binnen SBZ.

Reviewer: ```suggestion
De meetpunten worden om de zes jaar opgemeten. 
De ingezamelde gegevens laten toe om per habitattype een uitspraak te doen over het aandeel habitat dat zich in een gunstige/ongunstige toestand bevindt  in Vlaanderen en binnen SBZ.
```

Voor meer info omtrent de werkwijze van deze bepaling verwijzen we naar @paelinckx2019.

### Toepassingsgebied

> Dit veldprotocol is specifiek van toepassing bij de gegevensinzameling in het monitoringsmeetnet kwaliteit van de Natura2000 heidehabitats.

Reviewer: ```suggestion
Dit veldprotocol is specifiek van toepassing bij de gegevensinzameling in het meetnet biotische habitatkwaliteit van de Natura2000 heidehabitats.
```

Dit zijn in geologische termen jonge bodems die door zandverstuiving ontstaan zijn, soms bovenop oudere bodems, waar wel een profielontwikkeling heeft plaatsgevonden (podsols).
Op het terrein kan je dus naast de bodemkaart met behulp van een grondboor vaststellen of er profielontwikkeling is in het bovenste deel (bv. minimaal enkele tientallen cm).
Een landduinlandschap heeft vaak nog een duinstructuur, waarbinnen grotere en/of kleinere zandverstuivingen herkenbaar zijn (zie Figuur \@ref(fig:landduin)).
Dit duidt er dan in meer vastgelegde situaties op dat je je in een voormalig stuifzandgebied bevindt.
Vegetaties met dwergstruiken op landduinbodem resorteren onder 2310 psammofiele heide (psammos= zand in het Grieks) en de pioniersvegetaties (buntgras, zandstruisgras, mossen en korstmossen) worden als 2330 geclassificeerd.

(ref:landduin) Voorbeeld van een psammofiele buntgrasvegetatie (2330_bu) op stuifzandbodem, met een typisch duinreliëf, doch in een gefixeerd stadium (weinig open zand)

```{r landduin, fig.cap="(ref:landduin)"}
include_graphics(
  path = file.path(path_to_protocol,
                   "media",
                   "stuifduin.jpg"))
```

> **2330_bu buntgrasvegetatie op landduinen** Buntgrasvegetaties komen voor op landduinen waar de bodem door de hoge dynamiek nog geen profielontwikkeling heeft doorgemaakt (jonge bodems).Hoogstens is er een dunne humuslaag aanwezig.In goed ontwikkelde vormen zijn er aanzienlijke delen open zand maar de meeste landduinen in Vlaanderen zijn in min of meerdere mate gefixeerd door onvoldoende windwerking.

Reviewer: ```suggestion
**2330_bu buntgrasvegetatie op landduinen** Buntgrasvegetaties komen voor op landduinen waar de bodem door de hoge dynamiek nog geen profielontwikkeling heeft doorgemaakt (jonge bodems).
Hoogstens is er een dunne humuslaag aanwezig.
In goed ontwikkelde vormen zijn er aanzienlijke delen open zand, maar de meeste landduinen in Vlaanderen zijn in min of meerdere mate gefixeerd door onvoldoende windwerking.
```

De vegetatie wordt gekenmerkt door een hoog aandeel mossen en korstmossen die de bodem in een eerste fase fixeren, in combinatie met een aantal typische zandbindende vaatplanten.

*Typische soorten: Buntgras (Corynephorus canescens), Heidespurrie (Spergula morisonii), Zandzegge (Carex arenaria), Zandstruisgras (Agrostis vinealis), Zilverhaver (Aira caryophyllea), Vroege haver (Aira praecox), Dwergviltkruid (Filago minima), Ruig haarmos (Polytrichum piliferum), rendiermossen (Cladina spp.), heidestaartjes en bekermossen (Cladonia spp.) en Cetraria spp.(IJslands mos en Kraakloof)*

Op minder dynamische, schrale standplaatsen in heidegebieden kan je ook dwerghavervegetaties aantreffen (2330_dw).
Het onderscheid tussen beide subtypes is zelden echt scherp maar doorgaans zijn de dwerghavervegetaties wat minder rijk aan mossen en korstmossen, en wordt buntgras vervangen door Vroege en/of Zilverhaver.

> **2310 Droge heide op landduinen** Dit type heide komt voor op dezelfde types bodems als de buntgrasvegetaties maar omvatten een latere in de successiereeks waarbij de landduin verder gefixeerd is geraakt.

Reviewer: ```suggestion
**2310 Droge heide op landduinen** Dit type heide komt voor op dezelfde types bodems als de buntgrasvegetaties maar vormt een later stadium in de successiereeks, waarbij de landduin verder gefixeerd is geraakt.
```

Op de natste plekken spreken we van venige heide waar dopheide wat minder abundant is en het vooral helofyten en veenmossen zijn die aspectbepalend worden.
Ook door pijpenstrootje gedomineerde situaties op natte bodems kwalificeren als 4010.

*Typische soorten: Gewone dophei (Erica tetralix), Kleine zonnedauw (Drosera intermedia), Ronde zonnedauw (Drosera rotundifolia), Veenpluis (Eriophorum polystachion), Klokjesgentiaan (Gentiana pneumonanthe), Trekrus (Juncus squarrosus), Beenbreek (Narthecium ossifragum), Witte snavelbies (Rhynchospora alba), Veenbies (Scirpus cespitosus), Tweenervige zegge (Carex binervis), Bruine snavelbies (Rhynchospora fusca), Wilde gagel (Myrica gale) en qua veenmossen vooral Kussentjesveenmos (Sphagnum compactum), Wrattig veenmos (Sphagnum papillosum), Zacht veenmos (Sphagnum tenellum) maar ook gewoon veenmos (Sphagnum palustre) en geoord veenmos (Sphagnum auriculatum)*

**4030 Droge Europese heide** Groenblijvende formaties van dwergstruiken, en dan voornamelijk struikhei.

> Ook gedegradeerde (vergrast met pijpestrootje of bochtige smele en/of deels verboste) heideterreinen vallen hieronder.

Reviewer: soortnamen ofwel steeds met hoofdletter ofwel steeds zonder

Author: Ik doe een totale check

*Typische soorten: Struikhei (Calluna vulgaris), Klein warkruid (Cuscuta epithymum), Kleine wolfsklauw (Diphasiastrum tristachyum), Rode dophei (Erica cinerea), Stekelbrem (Genista anglica), Kruipbrem (Genista pilosa), Grote wolfsklauw (Lycopodium clavatum), Rode bosbes (Vaccinium vitis-idaea), blauwe bosbes (Vaccinium myrtillus)*

**Open vegetatietypes gebonden aan het heidelandschap die bij dit onderdeel van het meetnet worden gerekend maar dus mogelijk wel in complex met deze meetpunten voorkomen:**

> *7140_oli Overgangsvenen*

Reviewer: ```suggestion
**7140_oli Overgangsvenen**
```

Dit zijn venige heides of verlandende vennen waar de veenvorming al dermate doorgezet is dat er hoogveensoorten beginnen te groeien.
Ze luiden een relatief lange overgangsfase in naar *actief hoogveen (7110)*, dat in Vlaanderen enkel nog aan het Ven onder de Berg voorkomt.
Specifieke veenmossoorten (vaak typische met wijnrode tinten) zijn hier indicatief naast hoogveensoorten als Lavendelheide, Eénarig wollegras en Kleine veenbes.
Er is een afwisseling van veenmosbulten met waterhoudende depressies met venachtige begroeiingen.

*Typische soorten: Lavendelhei (Andromeda polifolia), Eénarig wollegras (Eriophorum vaginatum), Kleine veenbes (Vaccinium oxycoccos), Ronde zonnedauw (Drosera rotundifolia), Witte snavelbies (Rhynchospora alba), Slijkzegge (Carex limosa), Veenbloembies (Scheuchzeria palustris) , Veenmosorchis (Hammarbya paludosa), Hoogveenveenmos (Sphagnum magellanicum), Wrattig veenmos (Sphagnum papillosum), Rood veenmos (Sphagnum rubellum), aangevuld met alle typische soorten van 4010.*

*7150 Snavelbiesvegetaties*

Pioniersgemeenschappen met snavelbies op plagplekken of ven(oever)situaties met schommelende waterstanden.

> Meestal neemt dit type slechts geringe oppervlaktes in en evolueert het op plagplekken na een aantal jaar verder naar natte heide waarmee het doorgaans in complex voorkomt.Door betreding kunnen deze situaties ook in stand gehouden worden.De kruidlaag is ijl met een aanzienlijk aandeel open bodem maar kan in latere fasen hoge bedekkingen aan natte heidesoorten kennen.

Reviewer: ```suggestion
Meestal neemt dit type slechts geringe oppervlaktes in en evolueert het op plagplekken na een aantal jaar verder naar natte heide waarmee het doorgaans in complex voorkomt.
Door betreding kunnen deze situaties ook in stand gehouden worden.De kruidlaag is ijl met een aanzienlijk aandeel open bodem maar kan in latere fasen hoge bedekkingen aan natte heidesoorten kennen.
```
